models/calendar.yml

```diff
@@ -359,6 +359,19 @@
             exclude: true
 
     - 8:45 PM: Speaker TBA
+        type: Talk
```

Comment:
Oop, looks like you've got an indentation error! Would you mind adding another space here and to all following lines in your entry?